src/Templates/Microsoft.CodeAnalysis.AnalyzerTemplate/MyProject.Analyzer.CodeFixes/CodeFixResources.Designer.cs

```diff
@@ -0,0 +1,84 @@
+//------------------------------------------------------------------------------
```

Comment:
For dotnet/roslyn-analyzers repo, `*.Designer.cs` are only produced in artifacts. I don't know how this is done (probably just not including them in cspoj?). But generally I think it makes sense for the template to do the same if it's not a complicated thing to do.